lib/OpenQA/Git.pm

```diff
@@ -5,8 +5,12 @@ package OpenQA::Git;
 
 use Mojo::Base -base, -signatures;
 use Mojo::Util 'trim';
+use Mojo::URL;
 use Cwd 'abs_path';
-use OpenQA::Utils qw(run_cmd_with_log_return_error);
+use Mojo::File 'path';
+use File::Basename;
```

Comment:
Can be replaced by Mojo::File method

Reply:
how?

Reply:
with https://github.com/os-autoinst/openQA/pull/6190#discussion_r1972395299

Reply:
ah